packages/shared/src/components/modals/Prompt.tsx

```diff
@@ -35,7 +35,7 @@ export function PromptElement(props: Partial<ModalProps>): ReactElement {
       size={promptSize}
       onRequestClose={onFail}
       className={className.modal}
-      overlayClassName="!z-[100]"
+      overlayClassName="!z-max"
```

Comment:
Why do we not use modal here?

Reply:
Not really sure, probably as it should be way above the modal when confirming to close a modal.

Reply:
Alright will test out if i find any weird stuff once merged